classes/license_image.bbclass

```diff
@@ -0,0 +1,281 @@
+ROOTFS_LICENSE_DIR = "${IMAGE_ROOTFS}/usr/share/common-licenses"
```

Comment:
>If I understand correctly, creating a copy of the bbclass in our layer is the solution when you need to change an upstream bbclass.

Do you have a source for that? I think that logic makes sense if we are trying to *extend* the behavior of the core layers. In our images, we would inherit the core layer's bbclass, and then inherit the meta-nilrt bbclass (or includes). But, if we're trying to *change* core layer behavior, I think maintaining a wholesale-copy of the core layer bbclass would be inefficient.

Also, it makes diffing our changes hard. :)

So I think I would prefer for these changes to live as a commit against the OE-core bbclass - which will also make it easier to upstream.